am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessRecord.java

```diff
@@ -15,7 +15,7 @@
 
     private final Set<Permission> explicitPermissions;
 
-    @Builder // All args constructor is needs for builder. @SuperBuilder cannot be used because IDE does not support it
+    @Builder // All args constructor is needed for builder. @SuperBuilder cannot be used because IDE does not support it
```

Comment:
Good spot. Have you checked this is the only place with spelling issue?

Reply:
👍